### PR TITLE
Create 2022-11-10-nvme-monitoring.md

### PR DESCRIPTION
**Pull request by shyamvalsan**
NVMe monitoring blog/guide/landing page.

### CODE REVIEW

File: blog/2022-11-10-nvme-monitoring.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+     type: System
+component: Disk
+   lookup: max -30s unaligned
+    units: state
+    every: 10s
+     crit: $this != nan AND $this != 0
+    delay: down 5m multiplier 1.5 max 2h
+     info: NVMe device $label:device has critical warnings
+       to: sysadmin
+
```
Comment (ilyam8) [inline blog/2022-11-10-nvme-monitoring.md:139]:
```suggestion

```
 template: nvme_device_critical_warnings_state
 families: *
       on: nvme.device_critical_warnings_state
    class: Errors
     type: System
component: Disk
   lookup: max -30s unaligned
    units: state
    every: 10s
     crit: $this != nan AND $this != 0
    delay: down 5m multiplier 1.5 max 2h
     info: NVMe device $label:device has critical warnings
       to: sysadmin
```

```

File: blog/2022-11-10-nvme-monitoring.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+    class: Errors
+     type: System
+component: Disk
+   lookup: max -30s unaligned
+    units: state
+    every: 10s
+     crit: $this != nan AND $this != 0
+    delay: down 5m multiplier 1.5 max 2h
+     info: NVMe device $label:device has critical warnings
+       to: sysadmin
```
Comment (ilyam8) [inline blog/2022-11-10-nvme-monitoring.md:138]: need to be changed to code block

```
template: nvme_device_critical_warnings_state
```

Comment (ilyam8) [reply blog/2022-11-10-nvme-monitoring.md:138]: If the description is not enough (you have it in the first sentence - it checks if there are any critical warnings) and we need to have the alarm details here.

Comment (shyamvalsan) [reply blog/2022-11-10-nvme-monitoring.md:139]: Kept the template in there so users can get a feel of what they have to edit/update. 

Changed to code block.

File: blog/2022-11-10-nvme-monitoring.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
+---
+slug: nvme-monitoring
+title: "How to monitor NVMe metrics servers?"
+description: "How to monitor NVMe web servers?"
+image: ![image](https://user-images.githubusercontent.com/24860547/201093559-35910c86-43c7-44d5-9b0f-ee493b752a57.png)
+tags: [how-to,infrastructure-monitoring,nvme,ssd,disk]
+keywords: [how-to,infrastructure-monitoring,nvme,disk,ssd, Non-Volatile Memory Host Controller Interface]
+authors: shyam
+---
+Use Netdata to effectively monitor and troubleshoot the performance of SSD disks that rely on NVMe (Non-Volatile Memory express) in your infrastructure. Preempt disk failures and take action to ensure your systems run without a glitch.
```
Comment (sashwathn) [inline blog/2022-11-10-nvme-monitoring.md:10]: Use Netdata to effectively monitor and troubleshoot the performance of NVMe (Non-Volatile Memory express) disks in your infrastructure. Preempt disk failures and take action to ensure your systems run without a glitch.

**Review by sashwathn — CHANGES_REQUESTED** [review]
I think referring to them as NVMe disks is sufficient especially in the intro section. You are anyways covering more details in the "What is NVME?" section.

File: blog/2022-11-10-nvme-monitoring.md (hunk 3 above)
Comment (shyamvalsan) [reply blog/2022-11-10-nvme-monitoring.md]: done.

**Comment by andrewm4894** [comment]
@shyamvalsan all the charts are kinda just straight lines. Which is probably usual or normal for this i guess. I'm probably over thinking it but wonder if we can spice them up in some way eg take the screenshots during some sort of interesting time or something.

**Comment by shyamvalsan** [comment]
> @shyamvalsan all the charts are kinda just straight lines. Which is probably usual or normal for this i guess. I'm probably over thinking it but wonder if we can spice them up in some way eg take the screenshots during some sort of interesting time or something.

Yeah, had the same thought - it is hard to create NVMe errors just to take some screengrabs though :D 

I'll be replacing the screenshots after [this change](https://github.com/netdata/charts/pull/154) goes live so hopefully the charts look a little more exciting.

**Review by sashwathn — APPROVED** [review]

**Pull request merged**